jekyll/_cci2/monitoring.adoc

```diff
@@ -14,10 +14,14 @@ toc::[]
 
 To enable system and Docker metrics forwarding to either AWS Cloudwatch or Datadog, navigate to your CircleCI Management Console, select Settings from the menu bar and scroll down to enable the provider of your choice (your-circleci-hostname.com:8800/settings#cloudwatch_metrics).
 
+// Can this just say 'metrics' rather than system and Docker metrics?
+
 === VM Service and Docker Metrics
 
 VM Host and Docker services metrics are forwarded via https://github.com/influxdata/telegraf[Telegraf], a plugin-driven server agent for collecting and reporting metrics.
 
+// what is meant by VM Host here? Does this actually mean metics for the Services Machine?
```

Comment:
I think we really mean Vm service. Hard to say why we drew the distinction between VM Host and Vm Service. Might be fine to just say services machine here.

Reply:
oK so the section heading is VM Service and Docker Metrics - for now I'm just going to change the text to say VM Service rather than VM Host for consistency, but let me know it ti should be Services Machine.